firefox-ios/firefox-ios-tests/Tests/XCUITests/ShareMenuTests.swift

```diff
@@ -4,6 +4,8 @@
 
 import Foundation
 
+let pdfUrl = "https://s28.q4cdn.com/392171258/files/doc_downloads/test.pdf"
```

Comment:
can we use one of the pdf files that we have in our [gcp bucket](https://console.cloud.google.com/storage/browser/mobile_test_assets/public?inv=1&invt=AbnnFw&project=moz-mobile-tools&supportedpurview=project&pageState=(%22StorageObjectListTable%22:(%22f%22:%22%255B%255D%22))) better?